css/includes/components/itilobject/_timeline.scss

```diff
@@ -287,7 +287,8 @@
         }
     }
 
-    &.Log {
+    &.Log,
+    &.ITILAutoBump {
```

Comment:
A suggestion regarding the new line in the timeline as it displays in the same way as logs entries.
For both we may reduce vertical spacing.
In the `.timeline-header` rule below (l318), i suggest to override `margin-bottom` to 0 value

Ex, from:

![image](https://github.com/glpi-project/glpi/assets/418844/60564f78-4394-4fee-a9ac-ceb6a0e0d401)

to 

![image](https://github.com/glpi-project/glpi/assets/418844/6ea38944-9af0-43f3-b01e-eac1cf1f9ef6)

This saves vertical space and ease reading imo